src/js/Cindy.js.wrapper

```diff
@@ -1,5 +1,7 @@
 /* CindyJS - (C) 2014-2015  J. Richter-Gebert
```

Comment:
Shouldn't this be "The CindyJS Project" instead of "J. Richter-Gebert"? Or at least "Jürgen Richter-Gebert"?